Makefile

```diff
@@ -39,28 +39,27 @@ lint_shellcheck:
 	find . -name '*.sh' -print0 | xargs -0 $(SHELLCHECK)
 
 .PHONY: dev
-dev: check-env-vars set_env_class_dev deploy_pipelines ## Deploy Pipelines to Dev Environment
-
-.PHONY: dev-bootstrap
-dev-bootstrap: check-env-vars set_env_class_dev vagrant-deploy ## Start DEV bootsrap
+dev: check-env-vars set_env_class_dev ## Set environment to DEV
 
 .PHONY: ci
-ci: check-env-vars set_env_class_ci deploy_pipelines  ## Deploy Pipelines to CI Environment
-
-.PHONY: ci-bootstrap
-ci-bootstrap: check-env-vars set_env_class_ci vagrant-deploy  ## Start CI bootsrap
+ci: check-env-vars set_env_class_ci ## Set environment to CI
 
 .PHONY: stage
-stage: check-env-vars set_env_class_stage deploy_pipelines  ## Deploy Pipelines to Staging Environment
-
-.PHONY: stage-bootstrap
-stage-bootstrap: check-env-vars set_env_class_stage vagrant-deploy  ## Start Staging bootsrap
+stage: check-env-vars set_env_class_stage  ## Set Envirnoment to Staging
 
 .PHONY: prod
-prod: check-env-vars set_env_class_prod deploy_pipelines  ## Deploy Pipelines to Production Environment
+prod: check-env-vars set_env_class_prod ## Set Envirnoment to Production 
+
+.PHONY: bootstrap
+bootstrap: vagrant-deploy ## Start bootsrap 
```

Comment:
We don't need a separate target for `vagrant-deploy` anymore. The contents could be moved into this target like `bootstrap-destroy`.